src/components/PostListLoader.tsx

```diff
@@ -0,0 +1,18 @@
+import { Skeleton, VStack } from "@chakra-ui/react";
+import React from "react";
+
+interface PostListLoaderProps {
+  blogPostPerPage: number;
+}
+
+export const PostListLoader = ({ blogPostPerPage }: PostListLoaderProps) => {
+  return (
+    <VStack>
+      {new Array(blogPostPerPage).fill("").map((name, index) => (
```

Comment:
is `.fill("")` needed? Isn't `.fill()` enough?

Also, usually `_` is used to indicate useless variable, like `name` in this example. So something like `.map((_, index) => ...`